Serialization/Serialization/Serialization/BinarySerializer.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Serialization
+{
+    /// <summary>
+    /// This uses BinaryFormatter to serialize objects to binary.
+    /// </summary>
+    public class BinarySerializer : ISerializer
+    {
+        /// <summary>
+        /// This converts objects to binary.
```

Comment:
The XML commentary for this  method could be something like, "Serializes the specified `<see cref="ClothingArticleList{T}" />` into a binary representation and saves that representation to a file with the provided name." That would make it really easy for someone to understand and use your API. Also, the use of <see/> is nice because it allows linking back to the corresponding type (if the XML comments were converted to HTML using a 3rd party too) and the compiler would also rename the type here if you renamed the class.